variables.json

```diff
@@ -1,14 +1,14 @@
 {
   "containerd_version": "1.2.10-3",
   "docker_version": "5:18.09.9~3-0~ubuntu-bionic",
-  "kubelet_version": "1.18.10-00",
-  "kubeadm_version": "1.19.3-00",
-  "kubectl_version": "1.19.3-00",
+  "kubelet_version": "1.19.4-00",
+  "kubeadm_version": "1.19.4-00",
```

Comment:
We don't want a commit on master where the config is invalid.
Until 1.20.0 is out I think we should wait with this and only have it on branches for testing.

Reply:
Sure, the PR can wait till 1.20.0 is released!